sql/mysql/Positive-Technologies/MySqlParser.g4

```diff
@@ -2668,7 +2669,11 @@ bitOperator
     ;
 
 mathOperator
-    : '*' | '/' | '%' | DIV | MOD | '+' | '-'
+    : '*' | '/' | '%' | DIV | MOD
+    ;
+
+mathOperator2
+    : '+' | '-'
```

Comment:
I suggest renaming `mathOperator` to `multOperator` (short form of `multiplicative`), `matchOperator2` to `addOperator` (short form of `additive`)

Reply:
done